### PR TITLE
Add additional fields of interest.

### PR DESCRIPTION
**Pull request by Uberi**
Ref: https://bugzilla.mozilla.org/show_bug.cgi?id=1250889

### CODE REVIEW

File: src/main/scala/streams/Longitudinal.scala [hunk 1]
```diff
@@ -702,20 +707,17 @@ case class Longitudinal() extends DerivedStream {
...
-    val countries = payloads.map{ case (x) =>
-      x.getOrElse("geoCountry", "").asInstanceOf[String]
-    }
-    val cities = payloads.map{ case (x) =>
-      x.getOrElse("geoCity", "").asInstanceOf[String]
+  private def value2Avro[T:ClassTag](field: String, avroField: String, default: T,
+                          payloads: List[Map[String, Any]],
+                          root: GenericRecordBuilder,
+                          schema: Schema) {
+    val values = payloads.map{ case (x) =>
```
Comment (vitillo) [inline src/main/scala/streams/Longitudinal.scala]: See comments in #31.

**Comment by Uberi** [comment]
Changes:

* Cherry picked "Implement code review feedback" from `fix-dates`
* #31 is also updated

**Comment by Uberi** [comment]
Changes:

* Cherry-picked "Implement code review feedback" from `fix-dates`
* Update #31

**Comment by vitillo** [comment]
Looks good, I am waiting for @mreid-moz to comment on the bug.

**Comment by vitillo** [comment]
"The web server that receives Telemetry submissions stores the actual DNT header 
value the client sends. This is more comprehensive than the DNT_USAGE 
histogram, because it is included for all document types (even ones without 
histograms), and that histogram is only being collected for clients who opt in 
to extended telemetry. "

**Comment by Uberi** [comment]
The latest commit adds this field. `dnt_header` will be either the string `0`, `1`, or some other unknown value. As a result, it's a string. This field doesn't seem to be present in most telemetry pings from the pipeline.

**Comment by vitillo** [comment]
Looks good, thanks.

**Pull request merged**